src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/HearingResult.tsx

```diff
@@ -14,6 +14,15 @@ export const capitaliseExpression = (expression: string) => {
   return expression.charAt(0).toUpperCase() + expression.slice(1).toLowerCase()
 }
 
+export const getUrgentYesOrNo = (urgent: boolean | undefined): string => {
+  return urgent === true ? "Y" : "N"
+}
+
+export const getNumberOfHours = (hours: number | undefined): string | undefined => {
+  console.log(hours)
```

Comment:
Is this `console.log` still here intentionally?

Reply:
nope, no it is not